src/plugins/files/server/file_client/integration_tests/es_file_client.test.ts

```diff
@@ -239,11 +238,13 @@ describe('ES-index-backed file client', () => {
       })
     );
 
-    await esClient.indices.refresh({ index: blobStorageIndex });
+    // await esClient.indices.refresh({ index: blobStorageIndex });
```

Comment:
If the updated code version works, can we clean this up?

Reply:
done in https://github.com/elastic/kibana/pull/193792/commits/b42530a894a59c37147053c1c84c6b19dc4372fb